packages/shared/src/contexts/PaymentContext.tsx

```diff
@@ -87,9 +87,13 @@ export const PaymentContextProvider = ({
             break;
           case CheckoutEventNames.CHECKOUT_COMPLETED:
             logRef.current({
-              event_name: LogEvent.CompleteCheckout,
+              event_name:
+                'gifter_id' in event.data.custom_data
+                  ? LogEvent.GiftSubscription
```

Comment:
This is the wrong event. It should be `complete gift checkout`. I think we already have it in the enum